OpenRA.Game/Widgets/WidgetBounds.cs

```diff
@@ -0,0 +1,37 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Primitives;
+
+namespace OpenRA.Widgets
+{
+	public struct WidgetBounds
```

Comment:
do we want to extract structs as well?

Reply:
Oh, I guess not. I didn't see it was a struct.

Reply:
But in the end - why not extract it? It's reasonably complex.